airbyte-integrations/connectors/source-persistiq/source_persistiq/bootstrap.md

```diff
@@ -0,0 +1,18 @@
+# PersistIq
```

Comment:
This file is a level too deep, should rather be in `airbyte-integrations/connectors/source-persistiq/`